Favicons in Menu and Growl notification

netj opened the pull request:
Hi,

I've modified Reader Notifier Reloaded to show Favicon of each feed item's site in the menu and Growl notifications.  Please pull my work if you're interested!  I'm providing prebuilt binary of my version from my downloads page, but I would be happier if my enhancements get distributed via the auto-update feature. :)

Thanks,

~Jaeho

netj committed:
Removed the annoying "Unable to connect" alert.

 - I think it's better to show the alert only when the action was
   initiated by the user.  Internet connection may come and go, but
   Reader Notifier disturbs the user with an alert even though he/she is
   not interacting with it.

netj committed:
Updated English.lproj and added Korean.lproj.

 - Did `genstrings -o English.lproj Controllers/*.m` before adding
   Korean translations.
 - Localization Suite from http://www.loc-suite.org/ helped a lot.

netj committed:
gitignore, xcode project update

netj committed:
Favicons in Growl Notifications

 - Displaying the favicon of each feed item's linked site in Growl
   notifications.
 - Favicons are fetched all from Google's cache service,
   s2.googleusercontent.com instead of bothering each host.  The only
   problem with this approach is that all favicons, even though some
   small number of sites provide larger ones, become 16x16 which might
   not look so smooth with most of the Growl styles.

netj committed:
Polished nibs, added myself as a maintainer.

netj committed:
Fixed bugs in Korean nibs

A textfield label was blocking its popup button :(

netj committed:
Using Favicons in Menu

 - It seems showing favicons in the menu improves the overall usability a lot.

netj committed:
Fixed Shift-, Cmd- menus to have icons on them

netj committed:
Added my name to README

netj committed:
Improved favicons for feed proxies

 - FeedBurner and alikes obfuscate URLs identifiable with the host part
   being one among feedproxy.google.com, feeds.gawker.com, etc.
 - Now displays favicons in notifications even when minimal function.

netj committed:
Fixed feed proxy delegations

Commit:
Change URL for preview to match current Google

Commit:
Update to xcode 4

tyilo commented:
The favicon for iDownloadBlog (http://feeds.feedburner.com/iphonedlb) is the standard feedburner icon, not the icon of the blog.
In Reader Notifier Reloaded:
![Alt Text](http://f.cl.ly/items/213v1A3V2T1w0o1t0H2t/Screen%20Shot%202011-09-25%20at%2002.43.41%20.png)
In Google Reader:
![Alt Text](http://f.cl.ly/items/3D1t2w0S140r3e423s3X/Screen%20Shot%202011-09-25%20at%2002.44.15%20.png)

netj committed:
Merge remote-tracking branch 'alanshutko/master'

Conflicts:
	Reader Notifier.xcodeproj/project.pbxproj

netj committed:
Fixed feed quick add to use the correct API URL

As per:
https://sites.google.com/site/ursuleacv/knowledge/google-reader-api

netj committed:
Release 2.3, complete fork by netj

- Rewrote README (in Markdown).
- New place to distributed built versions. (file downloads are no longer
  supported at Github)
- Forked Sparkle appcast: new key, and URL pointing to my Dropbox.
- Wrote a small shell script for simplifying the process of creating zip
  archives, signing and deploying updates.
- Merged alanshutko's Google Reader URL fix.
- Growl framework 2.0.1.
  Notification Center support comes for free on OS X 10.8!
- Sparkle Korean language support.

netj committed:
Added screen shots to README

netj committed:
Fixed credits, comments, URLs, and version bump

to 2.3.1 to see if Sparkle updates actually work.
